python-client/docs/reference/models/integrations/base_classes.rst

```diff
@@ -0,0 +1,5 @@
+Base model classes
+==================
+
+.. automodule:: giskard.models.base
+    :members: BaseModel, WrapperModel, MLFlowSerializableModel, CloudpickleSerializableModel
```

Comment:
Could we rewrite this in a way that shows these 4 classes as a drop-down list when we click on "Base model classes" on the left toc menu?

Reply:
You mean the sidebar?

Reply:
yes, the left sidebar - not really urgent this one. I admittedly tried to do it myself, wasn't super intuitive without some files re-organization.